rasa/nlu/featurizers/count_vectors_featurizer.py

```diff
@@ -3,12 +3,13 @@
 import re
 from typing import Any, Dict, List, Optional, Text
 
+from sklearn.feature_extraction.text import CountVectorizer
```

Comment:
is this pycharm which rearranges all the imports?

Reply:
👍